massa-models/src/active_block.rs

```diff
@@ -2,16 +2,16 @@ use crate::{
     ledger_models::LedgerChanges,
     prehash::{Map, Set},
     rolls::RollUpdates,
-    Address, Block, BlockId, EndorsementId, OperationId,
+    Address, BlockId, EndorsementId, OperationId, Slot,
 };
 
 /// Block that was checked as final, with some useful precomputed data
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
 pub struct ActiveBlock {
     /// The creator's address
     pub creator_address: Address,
-    /// The block itself, as it was created
-    pub block: Block,
+    /// The id of the block
+    pub block: BlockId,
```

Comment:
I think this field should be renamed to block_id even if it implies a lot of changes in the codebase.